docs/ci-status.rst

```diff
@@ -35,6 +35,8 @@ Anaconda
 
 .. _Dependabot: https://github.com/rhinstaller/anaconda/network/updates
 
+.. _CockpitLibUpdate: https://github.com/rhinstaller/anaconda/actions/workflows/cockpit-lib-update.yml
+
```

Comment:
You want to do it like rest above, the Dependabot bit is the only different one because there's no standard status for its config parsing.

Reply:
Ouch, that was probably too little information. Gotta love ReST... You need two blocks of code for each of the badges on the resulting page. First, which you already got now, is a definition of the image. The second part (below) is where you actually use the image in the doc/page. Sorry about being unclear :(

Reply:
Woops :) Fixed it